tests/integration/mixer/scenarios_test.go

```diff
@@ -120,46 +121,58 @@ func testMetric(t *testing.T, ctx framework.TestContext, label string, labelValu
 
 // Port of TestTcpMetric
 func TestTcpMetric(t *testing.T) {
-	t.Skip("https://github.com/istio/istio/issues/11430")
-	ctx := framework.NewContext(t)
-	defer ctx.Done(t)
-
-	ctx.RequireOrSkip(t, environment.Kube)
-
-	d := bookinfo.DeployOrFail(t, ctx, bookinfo.BookInfo)
-	_ = bookinfo.DeployOrFail(t, ctx, bookinfo.BookinfoRatingsv2)
-	_ = bookinfo.DeployOrFail(t, ctx, bookinfo.BookinfoDb)
-
-	g := galley.NewOrFail(t, ctx, galley.Config{})
-	_ = mixer.NewOrFail(t, ctx, mixer.Config{Galley: g})
+	framework.Run(t, func(ctx framework.TestContext) {
+		ctx.RequireOrSkip(t, environment.Kube)
 
-	g.ApplyConfigOrFail(
-		t,
-		d.Namespace(),
-		bookinfo.NetworkingBookinfoGateway.LoadOrFail(t),
-		bookinfo.GetDestinationRuleConfigFile(t, ctx).LoadOrFail(t),
-		bookinfo.NetworkingVirtualServiceAllV1.LoadOrFail(t),
-		bookinfo.NetworkingTCPDbRule.LoadOrFail(t))
+		bookinfoNs, err := namespace.New(ctx, "istio-bookinfo", true)
+		if err != nil {
+			t.Fatalf("Could not create istio-bookinfo Namespace; err:%v", err)
+		}
+		d := bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Namespace: bookinfoNs, Cfg: bookinfo.BookInfo})
+		_ = bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Namespace: bookinfoNs, Cfg: bookinfo.BookinfoRatingsv2})
+		_ = bookinfo.DeployOrFail(t, ctx, bookinfo.Config{Namespace: bookinfoNs, Cfg: bookinfo.BookinfoDb})
 
-	prom := prometheus.NewOrFail(t, ctx)
-	ing := ingress.NewOrFail(t, ctx, ingress.Config{Istio: ist})
+		g := galley.NewOrFail(t, ctx, galley.Config{})
+		_ = mixer.NewOrFail(t, ctx, mixer.Config{Galley: g})
 
-	err := visitProductPage(ing, 30*time.Second, 200, t)
-	if err != nil {
-		t.Fatalf("unable to retrieve 200 from product page: %v", err)
-	}
+		ns, err := namespace.Claim(ctx, "default")
```

Comment:
Let's try to avoid using constant namespaces.  Can you use New instead of Claim here?

Reply:
Fixed and all tests are passing now! Can you please take a look?